web_app/api/user.py

```diff
@@ -298,3 +300,36 @@ async def allow_notification(
         raise HTTPException(status_code=404, detail=str(e))
     except Exception as e:
         raise HTTPException(status_code=500, detail="Internal server error")
+
+
+@router.post(
+    "/api/withdraw-all",
+    tags=["User Operations"],
+    summary="Withdraw all tokens from user's contract",
+    response_description="Status of withdrawal operations",
+)
+async def withdraw_all(wallet_id: str) -> dict:
+   """
+   Withdraws all supported tokens from the user's contract.
+
+   :param wallet_id: The wallet ID of the user.
+   :return: detail: "Successfully initiated withdrawals for all tokens"
+   """
+   # Get user's contract address
+   contract_address = await get_user_contract(wallet_id)
+   if not contract_address:
+       raise HTTPException(status_code=404, detail="Contract not found")
+
+   try:
+       # Perform withdrawals
+       results = await CLIENT.withdraw_all(contract_address)
+       return {
+           "detail": "Successfully initiated withdrawals for all tokens",
+           "results": results
+       }
+   except Exception as e:
```

Comment:
which exception you are going to catch here, if you added handling in `CLIENT.withdraw_all` and don't raise any Exception ?